lib/gridium/version.rb

```diff
@@ -1,3 +1,3 @@
 module Gridium
-  VERSION = "0.1.19"
+  VERSION = "0.1.20"
```

Comment:
This needs to be incremented one more time actually.  The version on rubygems is already 1.20.  I've got everything in sync again now.  This probably means you'll need to increment the gem version in STUI as well.